bundles/CoreBundle/Resources/translations/en.json

```diff
@@ -120,13 +120,15 @@
     "Please check your mailbox.": "Please check your mailbox.",
     "Login": "Login",
     "Submit": "Submit",
+    "Too many failed login attempts, please try again in %minutes% minutes.": "Too many failed login attempts, please try again in %minutes% minutes.",
```

Comment:
not used anywhere..

Reply:
Actually is used here
https://github.com/pimcore/pimcore/pull/14568/files?w=1#diff-f00318ba8a3ce0e489cdeb4aca080c40d86e511fa1a2336f5d4808fd210bae01R77

https://github.com/symfony/security-core/blob/c141337bc7451f9a9e464733f1e536bf38d1d2fb/Exception/TooManyLoginAttemptsAuthenticationException.php#L39

In 11.x it works with the domain `admin` changed to `security` but it didn't work on 10.x